### ISSUE TITLE
if characters["**"]:HasStatusEffect("####") no longer working 

### ISSUE THREAD

**Issue by rockysilver** [Masuzu/ZooeyBot#799]
Hello, this had been happening for a while now, the if characters["**"]:HasStatusEffect("####") suddenly stop working for me. 

```
  if characters["Io"]:HasStatusEffect("1034_23") then 

    characters["Io"]:UseSkill(1)

  end
```

Io (light) suppose to use her 1 when she has 3 stack of Mystic Vortex, I forgot which zooeybot update it was, but a while ago it stopped working.

```
 if characters["Tweyen"]:HasStatusEffect("4018") then 

    characters["Tweyen"]:UseSkill(3)

  end
```

This also happened to Song, I set it so she will use her 3 to paralysis when the buff Bring the Thunder is up, but Song just never cast 3.

My full scripts  for both characters:

```
if characters["Tweyen"] ~= nil then 
  characters["Tweyen"]:UseSkill(1)
  characters["Tweyen"]:UseSkill(2)
  if characters["Tweyen"]:HasStatusEffect("4018") then 
    characters["Tweyen"]:UseSkill(3)
  end
  characters["Tweyen"]:UseSkill(4) 
end

if characters["Io"] ~= nil then 
  characters["Io"]:UseSkill(2)
  if characters["Io"]:HasStatusEffect("1034_23") then 
    characters["Io"]:UseSkill(1)
  else
    if characters["Io"].charge_gauge >= 25 then
      characters["Io"]:UseSkill(3)
    end
  end
end
```

**Comment by Masuzu** [comment]
Hello,
Thank you very much for reporting this!
I identified the cause and a fix will be released within 24 hours.
I apologize for the inconvenience.

**Comment by Masuzu** [comment]
This is now addressed in v2.12.9
